### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'font_features'

### ISSUE THREAD

**Issue by vbd** [Kozea/WeasyPrint#1261]
```
  File "c:\Anaconda3\lib\site-packages\weasyprint\text.py", line 695, in setup
    attr = context.font_features.get(features)
AttributeError: 'NoneType' object has no attribute 'font_features'
```

Using  mkdocs-with-pdf with MkDocs and Material for MkDocs

Any idea where to look for?

in mkdocs.yml
```
theme:
    font:false
```

- no additional stylesheets defined

**Comment by liZe** [comment]
Hello, and thank you for this issue.

It really looks like this bug has been fixed a long time ago in 4729eac. Could you please check that your WeasyPrint version is up-to-date?

**Comment by vbd** [comment]
Thank you very much for the hint. 
pip install --upgrade weasyprint solved my problem.
I should have thought of this myself, please excuse the circumstances
